src/components/SignupForm/SignupForm.js

```diff
@@ -30,7 +30,8 @@ class SignupForm extends Component{
             const data = {
                 id : uid,
                 email,
-                fullName
+                fullName,
+                downloadURL: "https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/No_image_available.svg/600px-No_image_available.svg.png",
```

Comment:
This image should be taken from local assets and not from a online URL. This is a bad practice.

Reply:
I want to use this image for the users who haven't set their profile picture. This picture will be used everywhere in the project, like on Forum Screen. So, this picture is needed to be present in the firebase firestore for further applications. If I use this image from local assets, I won't be able to use it elsewhere as because I cannot fetch this image from firestore.

I guess using online Url for images is considered as a bad practice because you never know when the image Url link goes down when it becomes unavailable. So to overcome this problem I have uploaded this picture on firebase storage and used that URL so that the image URL link never gets unavailable, and by this, I can use it on other screens too.

Please let me know if this approach is fine. 😇

Reply:
I'm still not convinced with your idea here. The question I have is that why you can not use the image in the project assets. Once you build the project with the assets, it will be there and no need to fetch the image every time, which I personally seems like a good resource management.

Reply:
You can store a boolean in Firebase to check whether the user has provided a profile picture or not. That boolean can be used to filter the image. Correct me if this is can not be done.

Reply:
I am going to use this image in the chat screen. When there is no image provided by a user, then only this image (or any default image) will be used. So if I save this image in assets and use it from there, I won't be able to display the image in this chat screen as provided in prototype. So that's why I am using this URL, this URL is also saved in the firestore, and when user sends any message, then this picture will be accessed via firestore. 

<img src="https://user-images.githubusercontent.com/40908684/121886523-1d16ed00-cd33-11eb-96d4-bba4fd70ad6b.png" width="200" >

Reply:
Your logic is clear. One question, Is this URL going to be duplicated over and over or is it just this URL? And is it possible to do something like this -> The user does not updated his profile picture. And there is a variable associated with the profile to check whether the profile picture is updated or not. The image is in local assets folder. If the variable says that user has not updated the profile picture, use the image in assets folder, if not use the provided image. In either case the image should be in device local storage. Loading the same image from firebase is quite expensive when device user is considered. This is doable I think. If you think this is not doable, it's fine.

Reply:
Oh! Okay @shehand  You are right. This will surely reduce the firebase firestore reads. I have updated the code with this approach! Let me know if I did things in the right way! Thank you!